guides/common/modules/ref_iss-import-export-cheat-sheet.adoc

```diff
@@ -26,7 +30,7 @@
 |=========================================================
 |Intent | Command
 
-|Import to a Content View version | `hammer content-import version --organization="_My_Organization_" --path=’/var/lib/pulp/imports/dump_dir’`
-
-|Import to an Organization's Library| `hammer content-import library --organization="_My_Organization_" --path=’/var/lib/pulp/imports/dump_dir’`
+|Import to a Content View version | `hammer content-import version --organization="_My_Organization_" --path="/var/lib/pulp/imports/_dump_dir_"`
+|Import a Repository| `hammer content-import repository --organization="_My_Organization_" --path="/var/lib/pulp/imports/_dump_dir_"`
+|Import to an Organization's Library| `hammer content-import library --organization="_My_Organization_" --path="/var/lib/pulp/imports/_dump_dir_"`
```

Comment:
Maybe also find a nice word than "dump_dir".

Reply:
@maximiliankolb Pls see update and let me know if it works.

Reply:
Perfect.